client/my-sites/importer/newsletter/subscribers/paid-subscribers/map-plan.tsx

```diff
@@ -88,7 +86,17 @@ export function MapPlan( {
 						isSmallestUnit: true,
 						stripZeros: true,
 					} ) }
-					/{ plan.plan_interval } { active_subscriptions }
+					/{ plan.plan_interval }
+					{ plan.active_subscriptions && (
+						<>
+							{ ' • ' }
+							{ sprintf(
+								// translators: %d is number of subscribers
+								_n( '%d active subscriber', '%d active subscribers', plan.active_subscriptions ),
```

Comment:
`plan.active_subscriptions` is supposed to be boolean, so not sure what's going on here. Needs sorting out before sending string for translating.

https://github.com/Automattic/wp-calypso/blob/94899f83728b2f9d0717eb8d1fa7c298089b52a3/client/data/paid-newsletter/use-paid-newsletter-query.ts#L54-L63

Reply:
Removed the translation function for this string, we'll figure it outside this PR.